pkg/controllers/jobset_controller.go

```diff
@@ -113,16 +117,12 @@ func (r *JobSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	// If all jobs have succeeded, JobSet has succeeded.
 	// TODO(#120): ensure that the successful jobs counted are actually the ones that we expect JobSet to have
```

Comment:
I think you say in your description that this solves #120 so should we remove the comment?

Reply:
removed